### ISSUE TITLE
Messed Up SVGs

### ISSUE THREAD

**Issue by poudink** [poudink/Plastik#19]
Due to Qt's fucked SVG rendering (see https://bugs.kde.org/show_bug.cgi?id=448234), the SVG versions of some icons are just broken and I'm not too sure how I'm supposed to fix them. Easy way out would be to just get rid of them so it just uses the non-broken bitmaps instead, but SVGs are cool and I would much rather just get them properly working. For now, I'll just hide the broken SVGs until I figure out how to fix them.

Messed up icons:
bordertool

**Comment by poudink** [comment]
nope, karbon is a different issue. oh well, fixed terminal icon at least. also noticed a lot of other icons have slightly rectangular canvases, meaning they're probably getting slightly stretched. I'll be fixing those.

**Comment by poudink** [comment]
okular icon is messed up too

**Comment by poudink** [comment]
Found what the problem was with the remaining stretched icons. They had a "Non-uniform scale!" according to Inkscape. Not sure what that means, but fixing that fixed the stretching. Was also able to fix the Okular icon by exporting it as an optimized SVG in Inkscape. This leaves only the bordertool icon, which I was able to partially fix by exporting it from Karbon, but still looks a bit broken outside of Inkscape.
